### ISSUE TITLE
Bug Report: Low Battery Warning in the log

### ISSUE THREAD

**Issue by home-automate** [bloomkd46/homebridge-XfinityHome#28]
### Describe The Bug:

The following Error is logged for my Motion Sensor that is low on battery:
[28/06/2023, 13:28:18] [Xfinity Home] Livingroom Motion: Unknown trouble detected! 
[28/06/2023, 13:28:18] [Xfinity Home] Livingroom Motion: Please open an issue about this. 
[28/06/2023, 13:28:18] [Xfinity Home] Livingroom Motion: [
  {
    "description": "Unknown trouble",
    "name": "senPreLowBat",
    "criticality": false,
    "timeStamp": 1687724137875
  }
] 

This equates to the following message on the panel:
Livingroom Motion "Near Low Battery" (so it is like a pre-warning)

"The battery in your Livingroom Motion Sensor needs to be replaced soon"

### To Reproduce:

Sensor with Low Battery

### Expected behavior:

Output as a warning. Not sure what this would translate to in the HomeKit world.

### Logs:

```shell
[28/06/2023, 13:28:18] [Xfinity Home] Livingroom Motion: Unknown trouble detected! 
[28/06/2023, 13:28:18] [Xfinity Home] Livingroom Motion: Please open an issue about this. 
[28/06/2023, 13:28:18] [Xfinity Home] Livingroom Motion: [
  {
    "description": "Unknown trouble",
    "name": "senPreLowBat",
    "criticality": false,
    "timeStamp": 1687724137875
  }
]
```

### Config:

```shell
Homebridge Xfinity plugin 3.3.0
```

### Operating System:

Raspberian

### Node Version:

18.16.0

### NPM Version:

9.5.1

### Homebridge Version:

4.50.4

### Plugin Version:

3.3.0

**Comment by bloomkd46** [comment]
Fixed version 3.4.0, please let me know if it worked before closing this issue

**Comment by home-automate** [comment]
Worked beautifully! Thanks so much!

![image](https://github.com/bloomkd46/homebridge-XfinityHome/assets/108997613/42013c9a-631b-435b-961a-d7f840024179)
